### DISCUSSION TITLE
Recommended marker for sparse plots: ●

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1844] [Show and tell]
Hey,

For the last few months I've been toying around for different unicode characters to use for plots when there are too few points for the Braille glpyhs to be easily visible. After a few iterations, this is the one I like most: ●

Below is my script that toggles it. Basically I hit `*` whenever I'm in a plot that's a bit sparse.

```
@Canvas.api
def toggle_plot_asterisk(canvas: Canvas) -> None:
    if canvas.options.disp_canvas_charset == ' \u25cf':
        canvas.options.disp_canvas_charset = None
    else:
        canvas.options.disp_canvas_charset = ' \u25cf'
   
Canvas.addCommand('*', 'toggle-plot-asterisk', 'toggle_plot_asterisk()') 
```

**Comment by saulpw** [comment]
Neat, @cool-RR !  Where does `DEFAULT_DISP_CANVAS_CHARSET` come from?  I believe you should be able to use `None` to unset it to the default value, too.

**Reply by cool-RR** [reply]
You're right. I fixed the code above now.

**Comment by asteroidb612** [comment]
This is exactly what I've been looking for! Thank you for sharing this @cool-RR !
